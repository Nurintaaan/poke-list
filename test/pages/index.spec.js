import { shallowMount } from '@vue/test-utils'
import PokemonList from '@/pages/index.vue'

describe('Pokemon page', () => {
  test('should show data correctly', () => {
    const wrapper = shallowMount(PokemonList)
    expect(wrapper.element).toMatchSnapshot()
  })
})
