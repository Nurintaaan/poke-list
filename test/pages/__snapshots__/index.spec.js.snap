// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon page should show data correctly 1`] = `
<div
  class="flex flex-row"
/>
`;
